doc/nrf/includes/nfc_log_debug.txt

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+  If you are using debug messages in NFCT driver : and :kconfig:option:`CONFIG_LOG_MODE_IMMEDIATE` is enabled then the NFCT driver may not be working properly.
```

Comment:
Many drivers and subsystem may not be working properly if LOG_MODE_IMMEDIATE is enabled because this mode breaks their real time requirements.

Reply:
So it looks like this is a general issue and does not apply only to the NTCT driver.